website/management/commands/update_faiss.py

```diff
@@ -0,0 +1,57 @@
+from pathlib import Path
+
+from django.core.management.base import BaseCommand
+
+from website.bot import embed_documents_and_save, load_document, split_document
+
+
+class Command(BaseCommand):
+    help = "Update the FAISS database with new documents"
+
+    def handle(self, *args, **kwargs):
+        # Calculate the base directory
+        base_dir = Path(__file__).resolve().parents[3]
+
+        # Set the paths to the website directory, documents, and faiss_index directories
+        website_dir = base_dir / "website"
+        documents_dir = website_dir / "documents"
+        faiss_index_dir = website_dir / ""
```

Comment:
Would this work?
```suggestion
        faiss_index_dir = website_dir
```

Reply:
@JisanAR03 use os.path.join for logic's related to directory management, it makes it platform independent.